src/TipoffPackage.php

```diff
@@ -58,6 +58,24 @@ public function __construct(Package $package)
         $this->setBasePath($package->basePath);
     }
 
+    public function hasRoute(string $routeFileName): Package
+    {
+        if (config('tipoff.api.enabled')) {
+            return parent::hasRoute($routeFileName);
+        }
+
+        return $this;
+    }
+
+    public function hasRoutes(...$routeFileNames): Package
+    {
+        if (config('tipoff.api.enabled')) {
+            return parent::hasRoutes(...$routeFileNames);
+        }
+
+        return $this;
+    }
```

Comment:
Should these be `hasApiRoute` and `hasApiRoutes` to distinguish from the other routes the package may provide?

Reply:
Sure - we can do that.  I'll add in config and methods for web routes at the same time.  One sec.

Reply:
Thanks!